app/assets/javascripts/application.js

```diff
@@ -470,7 +470,7 @@ function toggle_input_group(item) {
 function reloadOnAjaxComplete(element) {
   $(element).indicator_hide();
   $('[rel="twipsy"]').tooltip();
-  $('select:not(.without_select2)').select2({ allowClear: true });
+  $('select:not(.without_select2)').select2({ allowClear: true, width: 'resolve' });
```

Comment:
Wen initiating Select2 it better to set {width: 'resolve'}, otherwise it can be not as the original select.

![strange_width](https://cloud.githubusercontent.com/assets/11807069/11828568/8deb8c30-a39e-11e5-8352-0787662c9119.png)